python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -376,6 +383,45 @@ def processResults(self, science, matchedTemplate, difference, sources, table,
 
         return measurementResults
 
+    def removeBadSources(self, diaSources):
```

Comment:
Oh, we should probably make this `_removeBadSources`: we're trying to make the "private" parts of Tasks more explicitly so.